index.js

```diff
@@ -56,13 +53,14 @@ class ProtomuxRpcClient extends ReadyResource {
 
   async close () {
     this._backoff.destroy()
-    this.rpc.destroy()
+    if (this.rpc) this.rpc.destroy()
     if (this._pendingRPC) this._pendingRPC.destroy()
     return super.close()
   }
 
   async _close () {
     if (this._connecting) await this._connecting // Debounce
```

Comment:
i think you should set suspended = false here also to avoid a later deadlock, wdyt

Reply:
I can't see a deadlock scenario, the only place where we don't do conditional checks on both `closing` and `suspended` in the same condition is in the `_makeRequest` function, where it's expected (since we wait to be resumed there)

Reply:
But I'm fine with setting `suspended = false` if it makes the code easier to understand obviously